src/integrations/airtable.ts

```diff
@@ -66,20 +66,24 @@ export const find = async (table: string, filterByFormula?: string, fields?: str
     });
 }
 
-// commenting out because this program should READ-ONLY
+export const update = async (table, data) => {
```

Comment:
This looks good - but is super dangerous because we don't have a staging env - so just be very careful

Reply:
It would be much safer to validate the schema of "Data" to ensure that only "Notes" can be edited right now

Reply:
You could use JOI to confirm a schema with all the properties as optional, but limited to only the defined properties

Reply:
You could also define "data" as an interface so you'd see typeerrors while developing if you ever accidentally dropped an additional key in

Reply:
I think right now it's already being validated with Joi in `controllers` with `bodySchema`, unless I'm doing it wrong??

<img width="469" alt="image" src="https://user-images.githubusercontent.com/8000986/139559905-481d3a39-aac4-46cc-8a15-addb88d354f5.png">